deployment/keystone/changeset/update_nodes.go

```diff
@@ -14,14 +15,31 @@ import (
 	"github.com/smartcontractkit/chainlink/v2/core/services/keystore/keys/p2pkey"
 )
 
+type MCMSConfig struct {
+	MinDuration time.Duration
+}
+
 var _ deployment.ChangeSet[*UpdateNodesRequest] = UpdateNodes
 
 type UpdateNodesRequest struct {
 	RegistryChainSel uint64
 	P2pToUpdates     map[p2pkey.PeerID]NodeUpdate
 
-	UseMCMS bool
+	// MCMSConfig is optional. If non-nil, the changes will be proposed using MCMS.
+	MCMSConfig *MCMSConfig
+}
+
+func (r *UpdateNodesRequest) Validate() error {
+	if r.P2pToUpdates == nil {
```

Comment:
do we want to check that `RegistryChainSel` is valid and exists using env.chain and also `chain_selectors.GetSelectorFamily(chainSel)` ?